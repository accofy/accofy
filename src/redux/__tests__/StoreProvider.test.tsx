import '@testing-library/jest-dom';

import { render } from '@testing-library/react';

import { StoreProvider } from '@/redux/StoreProvider';

describe('StoreProvider', () => {
  it('snapshot testing', () => {
    const { container } = render(
      <StoreProvider>
        <div>store provider</div>
      </StoreProvider>,
    );
    expect(container).toMatchSnapshot();
  });
});
