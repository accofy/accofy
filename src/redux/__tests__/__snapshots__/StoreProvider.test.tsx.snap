// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StoreProvider snapshot testing 1`] = `
<div>
  <div>
    store provider
  </div>
</div>
`;
